### ISSUE TITLE
market1501 got 91.7% accuracy of top 1 , but cuhk03 only got 66.6% 

### ISSUE THREAD

**Issue by jokerlc** [michuanhaohao/AlignedReID#2]
hi, why i only got  66.6% top 1 accuracy  on cuhk03 dataset with the same setting as martket1501 trainning ?   (python train_ml_alignedreid.py  -d cuhk03 -a resnet50 --test_distance global_local  --stepsize 150  --lr 0.0002   --save-dir ./log/resnet50-cuhk03-test1/  --train-batch 256  --eval-step 150  --max-epoch 300  --reranking )

**Comment by michuanhaohao** [comment]
@jokerlc CUHK03 updated its evaluation, which is different from original paper. So 66.6% top-1 is high performance for cuhk03. The softmax baseline is 47.5% mAP for "New" CUHK03.

> Use -d cuhk03 when running the training code. In default mode, we use new split (767/700). If you wanna use the original splits (1367/100) created by [13], specify --cuhk03-classic-split. As [13] computes CMC differently from Market1501, you might need to specify --use-metric-cuhk03 for fair comparison with their method. In addition, we support both labeled and detected modes. The default mode loads detected images. Specify --cuhk03-labeled if you wanna train and test on labeled images.

**Comment by jokerlc** [comment]
@michuanhaohao Thanks!

**Comment by michuanhaohao** [comment]
Done
